Tests/scripts/validate.sh

```diff
@@ -0,0 +1,7 @@
+if [ "${CHECK_BACKWARD}" = "true" ] ;
```

Comment:
need to forward the exit code from the sdk to the bash script (current state the script will always exit with 0
add `exit $?`